pep-0677.rst

```diff
@@ -1059,6 +1075,56 @@ Moreover, none of these ideas help as much with reducing verbosity
 as the current proposal, nor do they introduce as strong a visual cue
 as the ``->`` between the parameter types and the return type.
 
+Alternative Runtime Behaviors
+-----------------------------
+
+The hard requirements on our runtime API are that:
+
+- It must preserve backward compatibility with ``typing.Callable`` via
+  ``__args__`` and ``__params__``.
+- It must provide a structured API, which should be extensible if
+  in the future we try to support named and variadic arguments.
+
+Using a flat API
+~~~~~~~~~~~~~~~~
+
+Our runtime API mirrors the AST in that we distinguish the case
+where the arguments is ``Ellipses`` versus ``CallableArgumentsList``
```

Comment:
It's spelled "Ellipsis".

Also, do you mean the `Ellipsis` builtin, or a new type added to `types`?

Reply:
The builtin - I can update it. My earlier draft had a new type in `types` but I don't think that approach was popular